server/service/mdm.go

```diff
@@ -2125,6 +2125,17 @@ func (svc *Service) BatchSetMDMProfiles(
 		}
 	}
 
+	// // TODO(AP): Uncomment to enable activity logging for Android profiles once implemented
```

Comment:
Would it make sense to just remove it, as I've added it as part of the GitOps and as a way to avoid a merge conflict?

https://github.com/fleetdm/fleet/pull/32402/files#diff-4a26d067592e7583ec6cf08bae3de86411268b6fa9e7b9568fe9998aa8c6ab78R2148-R2156